examples/hosts/app-integration/external-data/README.md

```diff
@@ -24,6 +77,22 @@ TODO: Document the bot pattern and how it is surfaced in the example code.
 
 TBD
 
+### Concepts of data in this repository
```

Comment:
High-level feedback for this section: I don't know how much we need to refer to implementation details here. E.g. the fact that we use SharedMap_s to store the Draft / Saved data. It might be better to just describe what they represent, and let the reader explore the code if they want to see the particular implementation details.

@ChumpChief interested in your thoughts on this.

Reply:
Yeah - probably just go to the level of detail here so the reader can see all the puzzle pieces fitting together, which is prob no deeper than the public API surface.  For deeper documentation, probably put it in the class TSDoc for the respective classes.

Reply:
Sounds good! I'll take the details out!